### PR TITLE
Use Win32 API lists from buildtools

### PR DESCRIPTION
**Pull request by JohnChen0**
Remove the local copies of Win32 API lists, and use the lists
from buildtools instead. Fixes issue #13738.

### CODE REVIEW

File: dir.props [hunk 1]
```diff
@@ -71,6 +71,7 @@
   <!-- Enable the analyzers for this repo -->
   <PropertyGroup>
     <EnableDotnetAnalyzers Condition="'$(EnableDotnetAnalyzers)'==''">true</EnableDotnetAnalyzers>
+    <UseWin32Apis>true</UseWin32Apis>
```
Comment (gkhanna79) [inline dir.props]: Should we add the check for "Windows_NT" as well?

**Comment by JohnChen0** [comment]
@dotnet-bot test Innerloop Ubuntu14.04 Debug Build and Test please

File: dir.props (hunk 1 above)
Comment (weshaggard) [inline dir.props]: We were using OSGroup before we should continue to use that instead of OSEnvironment.

**Pull request merged**